contracts/factory/src/contract.rs

```diff
@@ -557,23 +557,30 @@ pub fn query_fee_info(deps: Deps, pair_type: PairType) -> StdResult<FeeInfoRespo
 ///
 /// * **_msg** is the object of type [`MigrateMsg`].
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, ContractError> {
+pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
     let contract_version = get_contract_version(deps.storage)?;
 
     match contract_version.contract.as_ref() {
         "astroport-factory" => match contract_version.version.as_ref() {
-            "1.0.0" => {
```

Comment:
We need to think how to properly migrate this one. We have different versions deployed on testnet and mainnet, changing this part will break migration in one of them

Reply:
It makes sense. I returned the migration for previous versions and made the MigrationMsg more universal.

Reply:
I agree